apps/studio/components/interfaces/Storage/AnalyticsBuckets/index.tsx

```diff
@@ -33,7 +34,7 @@ export const AnalyticsBuckets = () => {
   ) => {
     const url = `/project/${ref}/storage/analytics/buckets/${encodeURIComponent(bucketId)}`
     if (event.metaKey || event.ctrlKey) {
-      window.open(url, '_blank')
+      window.open(`${BASE_PATH}${url}`, '_blank')
```

Comment:
unrelated but i feel like this should be handled by `Link` - will look into this